### PR TITLE
[GLTF-Import] Files get ignored, when File ending is not lower case

### PR DESCRIPTION
**Pull request by cefoot**
## Overview
When trying to import GLTF or GLB files with upper case File ending f.e. "Video camera.GLB" the GlbAssetImporter Script get's triggerd but inside GltfEditorImporter.OnImportGltfAsset the file get's ignored because the file ending is not lower case.

### CODE REVIEW

File: Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/GltfUtility.cs [hunk 1]
```diff
@@ -55,7 +55,7 @@ public static async Task<GltfObject> ImportGltfObjectFromPathAsync(string uri)
 
             if (useBackgroundThread) { await BackgroundThread; }
 
-            if (uri.EndsWith(".gltf"))
+            if (uri.ToLower().EndsWith(".gltf"))
```
Comment (Troy-Ferrell) [inline Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/GltfUtility.cs]:
```suggestion
            if (uri.EndsWith(".gltf", StringComparison.OrdinalIgnoreCase))
```

File: Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/GltfUtility.cs [hunk 2]
```diff
@@ -67,7 +67,7 @@ public static async Task<GltfObject> ImportGltfObjectFromPathAsync(string uri)
                     return null;
                 }
             }
-            else if (uri.EndsWith(".glb"))
+            else if (uri.ToLower().EndsWith(".glb"))
```
Comment (Troy-Ferrell) [inline Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/GltfUtility.cs]:
```suggestion
            else if (uri.EndsWith(".glb", StringComparison.OrdinalIgnoreCase))
```

Comment (cefoot) [reply Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/GltfUtility.cs]: of course way better! thx for pointing this out and done in new commit

**Comment by cefoot** [comment]
Now using StringComparison enum instead of ToLower().

**Comment by keveleigh** [comment]
Thanks for the contribution!

**Pull request merged**